checkstyle.xml

```diff
@@ -109,7 +109,7 @@
                      value="Type name ''{0}'' must match pattern ''{1}''."/>
         </module>
         <module name="MemberName">
-            <property name="format" value="^[a-z][a-z0-9][a-zA-Z0-9]*$"/>
+            <property name="format" value="^[a-z][a-zA-Z0-9]*$"/>
```

Comment:
Why is this changed? This is part of the google guide afaik and forbids member names like `aThing`

Reply:
ah, xAxis and yAxis. Should probably just be changed to have the the `x|y` (`|z`?) exceptions from LocalVariableName